### ISSUE TITLE
Grid View Stopped Working

### ISSUE THREAD

**Issue by spencer-html** [Fugiman/google-meet-grid-view#343]
When you press grid view, it doesn't move the tiles like it should. I'm on MacOS and Chrome up to date, but it's been happening to people across all platforms!

**Comment by Jkarhani** [comment]
Yes it's not about windows or mac, google itself update itself & then this grid was broken.

**Comment by spencer-html** [comment]
Oh ok. Do you have any idea when this will be fixed?

**Comment by rsaurc** [comment]
Its possible to fix that, i have the same problem in mac

**Comment by spencer-html** [comment]
> Its possible to fix that, i have the same problem in mac

How would you fix it? Uninstalling and then reinstalling doesn't work.

**Comment by rsaurc** [comment]
I am missing the question mark. It was not a statement but a question of
whether there is any solution

Atte.
Rodrigo Saur Caces
E-Mail: rodrigo.saur@gmail.com

**Comment by spencer-html** [comment]
Ohh ok

**Comment by Eagle-Man** [comment]
From the Grid View (Fix) GitHub page:

> UPDATE 02/11/2021:
> It turns out that Google is testing an update of Meet that disables the Grid View extension. This is why sometimes it works and sometimes not (you see the boxes in a vertical bar). If you seriously need Grid View right now, you might consider refreshing a few times the browser tab (you should end up loading the old compatible version at a certain point).
> I am really sorry but I have to temporarily remove the extension from Chrome Web Store. In fact, updating Grid View would require a huge amount of work or may not be feasible at all. Please, remember that Google Meet introduced native grid view (up to 49 tiles).
